config/alpha.json

```diff
@@ -1,5 +1,8 @@
 {
   "project": "terra-bard-alpha",
   "orchestrationRoot": "https://firecloud-orchestration.dsde-alpha.broadinstitute.org",
-  "samRoot": "https://sam.dsde-alpha.broadinstitute.org"
+  "samRoot": "https://sam.dsde-alpha.broadinstitute.org",
+  "bigQueryProject": "broad-dsde-prod-analytics-dev",
+  "bigQueryDatasetId": "warehouse",
```

Comment:
I'm not sure if this is the right dataset for alpha to use (same with staging and perf)

Reply:
One thing that we slacked about: should we maybe take the approach of logging and setting up a sink?  I'm just worried that this will fail writing to BQ when there's more traffic

Reply:
I'll look into that! I think it could simplify the implementation here too